samples/quickstart-csharp/Controllers/HomeController.cs

```diff
@@ -2,21 +2,26 @@
 // Licensed under the MIT License.
 
 using System;
-using System.Net.Http;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 
 namespace QuickstartSampleWebApp.Controllers
 {
 	public class HomeController : Controller
```

Comment:
same comments as in ApiController
(null check on configuration object and specific ArgumentNullException message)